client/components/editor/index.vue

```diff
@@ -34,7 +34,6 @@
             :parent-id="activity.id"
             v-bind="getContainerConfig(type)" />
           <assessments v-if="showAssessments" />
-          <exams v-if="showExams" />
```

Comment:
I fully agree but this should be a separate PR since it is a drastic change and will require data migrations and discussions on how it should behave (potentially different from other basic containers).